sender-sqs/src/test/java/zipkin/sender/awssqs/AwsBufferedSqsSenderTest.java

```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package zipkin.reporter.awssqs;
```

Comment:
I'd call this `zipkin.reporter.sqs` as there are clones I think that aren't made by amazon. ack that the group id we publish will say aws, but I don't think there's a risk of name conflict with sqs or sns which will need clarifying. wdyt?